.claude/hooks/compose-commands-backup-2026-04-06.sh

```
@@ -0,0 +1,628 @@
+        return 0  # No CR instruction block — nothing to check
+    fi
+
+    # Resolve repo dynamically from git remote (not hardcoded)
+    local repo
+    repo="$(git remote get-url origin 2>/dev/null | sed 's/.git$//' | sed 's|.*github.com[:/]||' | tr '/' '_')"
+    if [[ -z "$repo" ]]; then
+        repo="$GITHUB_REPOSITORY"  # safe default
+    fi
+
+    # Get CR's latest formal review SHA and current PR head SHA
+    local cr_review_sha pr_head_sha gh_repo="jleechanorg/$(echo "$repo" | tr '_' '/')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)